### ISSUE TITLE
line 3 char 7495 - expected ')' - ie5.5/6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#145]
```
What steps will reproduce the problem?
1.  use any version of ie8.js
2.  load the page
3.  error warning

What is the expected output?
no error
What do you see instead?
error window
line 3 char 7495 - expected ')'

What version of the product are you using? On what operating system?

on ie8.js 2.0 any beta version / ie5.5 or ie6 / XP

Please provide any additional information below.

i dunno if it's normal but ur saying i don't need ie7 if im using ie8
but ie8 doesn't fix the box model on ie5.5

also this error doesn't happend if im using ie7

```

Original issue reported on code.google.com by `fun...@gmail.com` on 10 Jan 2009 at 1:56

**Comment by GoogleCodeExporter** [comment]
```
Please provide a sample page.
```

Original comment by `dean.edw...@gmail.com` on 8 Feb 2010 at 3:59
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
i found the culprit
just add this in your css using ie8js

#something input:not(#phone) {
    border-bottom-left-radius:10px;
    border-bottom-right-radius:10px;
    -moz-border-radius:0 0 10px 10px;
    -webkit-border-bottom-left-radius:10px;
    -webkit-border-bottom-right-radius:10px;
}

working fine with ie7js
probably :not() related
```

Original comment by `fun...@gmail.com` on 8 Feb 2010 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the test case. Re-opened.
```

Original comment by `dean.edw...@gmail.com` on 9 Feb 2010 at 12:23
- Changed state: **Re-open**

**Comment by GoogleCodeExporter** [comment]
```
or it's maybe related to the high specificity of the style rule
```

Original comment by `fun...@gmail.com` on 9 Feb 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:50
- Changed state: **Fixed**
